docs/superpowers/plans/2026-05-15-thermal-80mm-tax-summary.md

```
@@ -0,0 +1,441 @@
+    <col width="14" align="right">{{tax_amount_display}}</col>
+  </row>
+  {{/tax_summary}}
+```
+
+Leave the `{{#has_tax_summary}}` heading block immediately above it untouched.
+
+- [ ] **Step 4: Update the 80mm header comment**
+
+In the same file, in the top comment block's section list, replace these two lines (~lines 26-27):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)